department-of-veterans-affairs/caseflow
# AttorneyQualityReviewTask

Bumps #10691 

### Description
Added in the AttorneyQualityReview type

### Acceptance Criteria
- [x] JudgeQualityReviewTasks children are AttorneyQualityReviewTasks, not AttorneyTasks

### Testing Plan
1. Go to QR user page
2. Return to a Judge.
3. Become judge and go to the case
4. Return to Attorney
5. Become Attorney & confirm type.
6. Draft decision to judge
7. Confirm as Judge
8. Confirm case returns to QR.

After deploy, update the prod cases
